src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -690,6 +711,16 @@ private function validateForm(): void
                     $data['remove_from_search_index'] = true;
                 }
 
+                // just execute if the site is multi-language
+                if ($this->getContainer()->getParameter('site.multilanguage')) {
+                    // loop active languages
+                    foreach (BL::getActiveLanguages() as $language) {
+                        if ($language != BL::getWorkingLanguage()) {
+                            $data['hreflang_' . $language] = $this->form->getfield('hreflang_' . $language)->getValue();
```

Comment:
maybe only add it when the field is actually filled, and the same in the edit action